api/kv.go

```diff
@@ -238,3 +280,35 @@ func (k *KV) deleteInternal(key string, params map[string]string, q *WriteOption
 	res := strings.Contains(string(buf.Bytes()), "true")
 	return res, qm, nil
 }
+
+// Txn is used to apply multiple KV operations in a single, atomic transaction.
+// Note that Go will perform the required base64 encoding on the values
+// automatically because the type is a byte slice.
```

Comment:
I know you're working on docs here, but can you include a small example snippet for how the API would use a transaction here in the comment block?